internal/uidriver/js/ui_js.go

```diff
@@ -92,11 +92,40 @@ func (u *UserInterface) ScreenSizeInFullscreen() (int, int) {
 }
 
 func (u *UserInterface) SetFullscreen(fullscreen bool) {
-	// Do nothing
+	if !canvas.Truthy() {
+		return
+	}
+	if !document.Truthy() {
+		return
+	}
+	if fullscreen {
+		f := canvas.Get("requestFullscreen")
+		if !f.Truthy() {
+			f = canvas.Get("webkitRequestFullscreen")
+			return
+		}
+		f.Call("bind", canvas).Invoke()
+		return
+	}
+	f := document.Get("exitFullscreen")
+	if !f.Truthy() {
+		f = document.Get("webkitExitFullscreen")
+	}
+	f.Call("bind", document).Invoke()
+	return
 }
 
 func (u *UserInterface) IsFullscreen() bool {
-	return false
+	if !document.Truthy() {
+		return false
+	}
+	if !document.Get("fullscreenElement").Truthy() {
+		if !document.Get("webkitFullscreenElement").Truthy() {
+			return false
+		}
+		return false
```

Comment:
```
if !document.Get("fullscreenElement").Truthy() && if !document.Get("webkitFullscreenElement").Truthy() {
    return false
}
```
?

Reply:
Long past few days. My bad. Not on my A game at the moment.